rhoshift/utils/constants.py

```
@@ -1,120 +1,495 @@
+        if config.create_namespace or config.namespace not in ['openshift-operators']:
+            og_exists, og_info = cls.check_existing_operator_group(config.namespace, oc_binary)
+            if og_exists:
+                if og_info == "shared_namespace_multiple_groups":
+                    # Shared namespace with multiple groups - don't create another
+                    should_create_og = False
+                elif og_info and "Multiple" in og_info:
+                    # Multiple operator groups found in dedicated namespace - this will cause issues
+                    import logging
+                    logger = logging.getLogger(__name__)
+                    logger.warning(f"⚠️  {og_info} in namespace {config.namespace}. This may cause installation issues.")
+                    logger.warning(f"   Consider cleaning up duplicate operator groups before installation.")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)